docs/INSTALL-devel.rst

```diff
@@ -69,44 +69,44 @@ Start postgresql service::
 Part 2: Create a system user, database and database role.
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Setting up development environment with PostgreSQL backend becomes 
-easy with a dedicated user controlling the database as well as the 
+Setting up development environment with PostgreSQL backend becomes
+easy with a dedicated user controlling the database as well as the
 development environment.
 
-Let that username be `haas_dev`.
+Let that username be `hil_dev`.
 For simplicity we will use the same name for database and database role.
 
 Create a new user on your system::
-  
-  useradd --system -m -d /home/haas_dev -s /bin/bash -c "for HaaS development" haas_dev
 
-This will create a `haas_dev` user with following attributes::
-  
-  haas_dev:x:1002:1002:for HaaS development:/home/haas_dev:/bin/bash
+  useradd --system -m -d /home/hil_dev -s /bin/bash -c "for HIL development" hil_dev
 
-exact uid/gid may vary depending on your system. 
+This will create a `hil_dev` user with following attributes::
 
-Switch to the `haas_dev` user::
+  hil_dev:x:1002:1002:for HIL development:/home/hil_dev:/bin/bash
 
-  sudo -u haas_dev -i
+exact uid/gid may vary depending on your system.
+
+Switch to the `hil_dev` user::
+
+  sudo -u hil_dev -i
 
 Setup database and role to control it.
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Create database named `haas_dev` owned by user also named as `haas_dev`.
+Create database named `hil_dev` owned by user also named as `hil_dev`.
+
+1. Create a database role named `hil_dev` with priviledges to::
```

Comment:
we misspell "privileges" as "priviledges" several times in the code base, and while not the topic of this PR, it does bring everyone's attention to it